core/templates/dev/head/services/contextual/UrlService.js

```diff
@@ -34,10 +38,28 @@ oppia.factory('UrlService', ['$window', function($window) {
       return urlParts[1] === 'embed';
     },
     getPathname: function() {
-      return $window.location.pathname;
+      return this.getCurrentUrl().pathname;
+    },
+    getParamValuesAsList: function(queryTag) {
+      var url = this.getCurrentUrl().href.slice(
+        this.getCurrentUrl().href.indexOf('?') + 1).split('&');
+      var parentIdArray = [];
+      for (var i = 0; i < url.length; i++) {
+        var urlparam = url[i].split('=');
```

Comment:
var currentFieldName (to distinguish from the passed-in fieldName arg)

Reply:
Done